index.js

```diff
@@ -0,0 +1,11 @@
+// const { min } = require("lodash");
+/* eslint-disable */
```

Comment:
- [ ] Kindly enable linters for this file, you may only disable them for `luxon` since it's a 3rd party library, and changing the file will change the behavior of the functionality.